libs/components/list-builder-view-checklist/src/lib/modules/list-view-checklist/list-view-checklist.module.ts

```diff
@@ -6,6 +6,9 @@ import { SkyCheckboxModule } from '@skyux/forms';
 import { SkyListViewChecklistItemComponent } from './list-view-checklist-item.component';
 import { SkyListViewChecklistComponent } from './list-view-checklist.component';
 
+/**
+ * @deprecated List builder and its features are deprecated. Use repeater instead. For more information, see https://developer.blackbaud.com/skyux/components/repeater.
```

Comment:
Should this message match the one from `list-view-checklist.component.ts`?

Reply:
Yep. Fixed.